corehq/apps/accounting/tasks.py

```diff
@@ -187,7 +187,7 @@ def warn_active_subscriptions_per_domain_not_one():
         ).count()
         if active_subscription_count > 1:
             log_accounting_error("Multiple active subscriptions found for domain %s" % domain_name)
-        elif active_subscription_count == 0:
+        elif active_subscription_count == 0 and Domain.get_by_name(domain_name).is_active:
```

Comment:
it looks like we have a `domain/by_status` view. Not sure if you looked at using that instead of pulling all domains, then doing another db call for all of them that fall into this category?

Reply:
Didn't know about this, I'll take a look.